### DISCUSSION TITLE
Hunter (warlock, dk) Pet Scaling changes

### DISCUSSION THREAD

**Discussion by Felcaster** [azerothcore/azerothcore-wotlk#16955] [Guides & Tips]
Hello. How to change damage scaling values for Hunter's pets?
File data/Spell.dbc, spell-34902 (Hunter Pet Scaling 01) in "EffectAura_1_2_3" have auras:
29 SPELL_AURA_MOD_STAT
99 SPELL_AURA_MOD_ATTACK_POWER
13 SPELL_AURA_MOD_DAMAGE_DONE
How to change values of these auras? For example it transfer 22% of Attack Power from hunter to pet. I would like to change to 45%. At what file and what cell I can find it?

**Comment by duncrafter** [comment]
You should look at EffectBasePoints_X (X is the number of the effect). This field contains the base value of the spell.
Probably helpfull: https://github.com/stoneharry/WoW-Spell-Editor

**Comment by Felcaster** [comment]
In "data/Spell.dbc" for "spell-34902" "EffectBasePoints-1-2-3" are 0-0-0.
Spell-Editor  show only "Spell Effect 6-Apply_Aura", Apply Aura Name:
29 SPELL_AURA_MOD_STAT or 99 SPELL_AURA_MOD_ATTACK_POWER or 13 SPELL_AURA_MOD_DAMAGE_DONE
no suitable numeric values.
So question is still actual - How to change values of these auras?
